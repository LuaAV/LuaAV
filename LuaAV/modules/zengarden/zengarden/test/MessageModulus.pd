#N canvas 786 205 217 358 10;
#X obj 69 12 loadbang;
#X obj 28 13 bng 15 250 50 0 empty empty empty 17 7 0 10 -262144 -1
-1;
#X obj 112 325 print;
#X obj 112 288 mod;
#X msg 69 165 -47;
#X msg 41 217 61;
#X msg 28 241 7;
#X msg 55 191 -12;
#X msg 97 114 2;
#X msg 83 138 0;
#X obj 28 44 t b b b b b b b b b;
#X msg 139 68 3;
#X msg 125 88 11;
#X connect 0 0 10 0;
#X connect 1 0 10 0;
#X connect 3 0 2 0;
#X connect 4 0 3 1;
#X connect 5 0 3 1;
#X connect 6 0 3 0;
#X connect 7 0 3 0;
#X connect 8 0 3 1;
#X connect 9 0 3 0;
#X connect 10 0 6 0;
#X connect 10 1 5 0;
#X connect 10 2 7 0;
#X connect 10 3 4 0;
#X connect 10 4 9 0;
#X connect 10 5 8 0;
#X connect 10 6 3 0;
#X connect 10 7 12 0;
#X connect 10 8 11 0;
#X connect 11 0 3 1;
#X connect 12 0 3 0;
