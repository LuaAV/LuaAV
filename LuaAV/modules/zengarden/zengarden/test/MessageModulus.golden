[@ 0.000ms] print: 2
[@ 0.000ms] print: 2
[@ 0.000ms] print: 0
[@ 0.000ms] print: 35
[@ 0.000ms] print: 7
